### ISSUE TITLE
Cannot not send/receive Emails

### ISSUE THREAD

**Issue by tobibe** [zammad/zammad#1032]
### Infos:

* Used Zammad version: 1.5.0
* Used Zammad installation source: package
* Operating system: Ubuntu 16.04.02
* Browser + version: Firefox 53.0

Since Monday we cannot receive or send emails. 

If I click on edit in Channel->Email->Email-Accounts and change noting Zammad tells me that 16 Emails will be imported and deleted from server. If I click OK nothing happens and the emails still remain at the Exchange server. Until yesterday everything worked fine.

In Channel->Email->Email-Notification I got the Error: Can't use Channel::Driver::Smtp: #<EOFError: end of file reached>

![grafik](https://cloud.githubusercontent.com/assets/9305993/25608181/37c5a760-2f1a-11e7-975f-ea71565d50b2.png)

Our Emailconfiguration from database to connect to our Exchange server, but here we changed nothing.
--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
outbound: !ruby/hash:ActiveSupport::HashWithIndifferentAccess
  adapter: smtp
  options: !ruby/hash-with-ivars:ActionController::Parameters
    elements:
      host: Exchangeserver
      user: zammad
      password: password
      port: '587'
      domain: domain
      enable_starttls_auto: true
      openssl_verify_mode: none
    ivars:
      :@permitted: false

One Error entry in production.log
E, [2017-05-02T08:40:55.770722 #968] ERROR -- : ERROR: Transaction::Notification
E, [2017-05-02T08:40:55.770901 #968] ERROR -- : ERROR: #<RuntimeError: Can't use Channel::Driver::Smtp: #<EOFError: end of file reached>>
E, [2017-05-02T08:40:55.771148 #968] ERROR -- : ["/opt/zammad/app/models/channel.rb:250:in `rescue in deliver'", "/opt/zammad/app/models/channel.rb:229:in `deliver'", "/opt/zammad/lib/notification_factory/mailer.rb:111:in `send'", "/opt/zammad/lib/notification_factory/mailer.rb:164:in `notification'", "/opt/zammad/app/models/transaction/notification.rb:199:in `block in perform'", "/opt/zammad/app/models/transaction/notification.rb:93:in `each'", "/opt/zammad/app/models/transaction/notification.rb:93:in `perform'", "/opt/zammad/app/models/observer/transaction.rb:57:in `execute_singel_backend'", "/opt/zammad/app/models/transaction/background_job.rb:30:in `block in perform'", "/opt/zammad/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.7.1/lib/active_record/relation/delegation.rb:46:in `each'", "/opt/zammad/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.7.1/lib/active_record/relation/delegation.rb:46:in `each'", "/opt/zammad/app/models/transaction/background_job.rb:26:in `perform'", "/opt/zammad/app/models/transaction/background_job.rb:36:in `run'", "/opt/zammad/app/models/ticket.rb:189:in `block in process_pending'", "/opt/zammad/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.7.1/lib/active_record/relation/delegation.rb:46:in `each'", "/opt/zammad/vendor/bundle/ruby/2.3.0/gems/activerecord-4.2.7.1/lib/active_record/relation/delegation.rb:46:in `each'", "/opt/zammad/app/models/ticket.rb:180:in `process_pending'", "(eval):1:in `_start_job'", "/opt/zammad/app/models/scheduler.rb:90:in `eval'", "/opt/zammad/app/models/scheduler.rb:90:in `_start_job'", "/opt/zammad/app/models/scheduler.rb:73:in `block in start_job'"]
I, [2017-05-02T08:40:56.022115 #968]  INFO -- : Send notification to: yyy@xxx.com (from Notification Master <zammad@xxx.com>)
E, [2017-05-02T08:40:56.163478 #968] ERROR -- : Can't use Channel::Driver::Smtp: #<EOFError: end of file reached>
E, [2017-05-02T08:40:56.163680 #968] ERROR -- : []

How can I repair the email handling?

Greetings 
tobibe

**Comment by tobibe** [comment]
Hello,

found the reason. 

It was an email which could not be processed from email account. Sadly this was not shown in Email-Channel-Settings.
I found this by manually fetching emails from Zammad console by "Channel.fetch". It throws some Errors, which I have regrettably not saved.
This mail completly destroyed all mail handling. No emails receiving or sending. And even no notifications.

I removed the email from the email account and restarted the zammad server. Now everything works fine again.

Greetings 
tobibe
